plugin-script/src/main/java/io/kestra/plugin/scripts/exec/scripts/services/ScriptService.java

```diff
@@ -18,11 +20,16 @@ private ScriptService() {
     }
 
     public static String replaceInternalStorage(RunContext runContext, @Nullable String command) throws IOException {
-        return io.kestra.core.models.script.ScriptService.replaceInternalStorage(runContext, command);
+        return io.kestra.core.models.script.ScriptService.replaceInternalStorage(runContext, command, (ignored, file) -> {}, false);
     }
 
     public static List<String> uploadInputFiles(RunContext runContext, List<String> commands) throws IOException {
-        return io.kestra.core.models.script.ScriptService.uploadInputFiles(runContext, commands);
+        try {
+            return io.kestra.core.models.script.ScriptService.replaceInternalStorage(runContext, Collections.emptyMap(), commands, (ignored, file) -> {}, false);
+        } catch (IllegalVariableEvaluationException e) {
+            // Throw unchecked exception to prevent breaking the old method signature
+            throw new RuntimeException(e);
+        }
     }
```

Comment:
Not mandatory but better to create a new method that declare throwing this exception then make the existing deprecated and using the new method